src/api/application/application.ts

```diff
@@ -42,13 +42,23 @@ export default class ApplicationModule extends Base {
     /**
      * Returns an Application object that represents an existing application.
      * @param { Identity } indentity
-     * @return {Promise.<Application>}
+     * @return Promise<Application>
      * @tutorial Application.wrap
      */
     public wrap(identity: Identity): Promise<Application> {
         return Promise.resolve(new Application(this.wire, identity));
     }
 
+    /**
+     * Returns an Application object that represents an existing application.
```

Comment:
Maybe mention 'synchronous' somewhere in the description?  (and add 'asynchronous' to other one)